src/GroupManager.php

```diff
@@ -121,13 +137,59 @@ public function removeGroup($entity_type_id, $bundle_id) {
       $editable->set('groups', $groups);
       $saved = $editable->save();
 
+      // Remove all roles associated with this group type.
+      $this->removeRoles($entity_type_id, $bundle_id);
+
       $this->refreshGroupMap();
 
       return $saved;
     }
   }
 
   /**
+   * Creates default roles for the given group type.
+   *
+   * @param string $entity_type_id
+   *   The entity type ID of the group for which to create default roles.
+   * @param string $bundle_id
+   *   The bundle ID of the group for which to create default roles.
+   */
+  protected function createDefaultRoles($entity_type_id, $bundle_id) {
+    $properties = [
+      'group_type' => $entity_type_id,
+      'group_bundle' => $bundle_id,
+    ];
+    foreach ([OgRoleInterface::ANONYMOUS, OgRoleInterface::AUTHENTICATED, OgRoleInterface::ADMINISTRATOR] as $role_name) {
+      // @todo: Find a way to deal with potential ID collisions.
```

Comment:
the id should be auto increment, and the name should be the $role name. From D7:

![_mysql_5_6_22__127_0_0_1_d7_dev_og_role](https://cloud.githubusercontent.com/assets/125707/15352768/83800612-1cee-11e6-9e1b-57eb021eb9e1.jpg)